### PR TITLE
Update for version 18

### PR DESCRIPTION
**Pull request by david-a-wheeler**

### CODE REVIEW

**Comment by david-a-wheeler** [comment]
The second commit makes the real changes in the documentation file (the first commit just records the older v17 documentation).

**Comment by david-a-wheeler** [comment]
@expln - I'm back from a work trip, here is the updated documentation!

File: docs/index.md [hunk 1]
```diff
@@ -4336,6 +4559,35 @@ Here is the equivalent JSON for it:
...
 
+#### Version 18
+
+This version added [transformers (was: New fragment selector to insert and elide statements)](https://github.com/expln/metamath-lamp/issues/121#issuecomment-1697997262)
+
+In many cases one statement is a common transformation of another
+statement. To make this easier to do (and more likely to be correct),
+a new fragment selector has been created to let you apply simple
+transforms to insert and elide statements.
+Some small changes are expected to occur in this before a release.
```
Comment (expln) [inline docs/index.md:4570]: ```Some small changes are expected to occur in this before a release.``` 

Looks like this is not needed anymore.

**Review by expln — APPROVED** [review]
Great! Thank you as always, David!

File: docs/index.md (hunk 1 above)
Comment (david-a-wheeler) [reply docs/index.md]: Agreed!

**Pull request merged**